core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4163,11 +4171,15 @@ protected function parentView()
 			$blnClipboard = true;
 			$arrClipboard = $arrClipboard[$table];
 
-			if (\is_array($arrClipboard['id']))
+			if (!empty($arrClipboard['id']) && \is_array($arrClipboard['id']))
```

Comment:
Could be shortened to this:
```suggestion
			if (\is_array($arrClipboard['id'] ?? null))
```
(`??` works on arrays as well as on `null`)

Reply:
Yeah, but there is a subtle difference:

```php
$arrClipboard['id'] = [];

if (\is_array($arrClipboard['id'] ?? null)) {
    // will be executed
}

if (!empty($arrClipboard['id']) && \is_array($arrClipboard['id'])) {
    // will not be executed
}
```

And since we do not want `$blnMultiboard` to be `true` if the array is empty, we need both checks here.

Reply:
On the other hand, we have always set `$blnMultiboard` to `true` even if the array was empty, so my change actually implies a behavior change. We should merge your suggestion instead. 👍🏻